r/tests/testthat/test-util.R

```diff
@@ -39,3 +39,34 @@ test_that("as_writable_table() errors for invalid input", {
   # make sure other errors make it through
   expect_snapshot_error(wrapper_fun(data.frame(x = I(list(1, "a")))))
 })
+
+test_that("all_funs() identifies namespace-qualified and unqualified functions", {
+  expect_equal(
+    arrow:::all_funs(rlang::quo(pkg::fun())),
```

Comment:
You don't need `arrow:::`, testthat runs in the package namespace

```suggestion
    all_funs(rlang::quo(pkg::fun())),
```

Reply:
💯 Oversight on my side

Reply:
Done. Removed.